script/setup-dev

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+sudo apt install -y curl python
+
+# node and npm and yarn and pnpm
+# https://github.com/nodesource/distributions/blob/master/README.md#debmanual
+NODE_VERSION=node_12.x
```

Comment:
@eventualbuddha I copied how setup-node works in vxsuite-complete-system here, it seems to work fine, but I install node 12.22.x instead of 12.9.0, do you know if thats fine? I think thats what the production machines are doing.

Reply:
We should probably use the latest 12.x, so whatever we have now that points to 12.9.0 should be updated.

That said, Node.js 12 only has another couple months of support, after which we'll be on our own. I recommend we prioritize an update to 14 (EOL 4/30/23) or 16 (EOL 4/30/24) ASAP.

Reply:
Oh good to know, I'll make a task for that.